app/static/buttons.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Author: Micah Weiberg
+ * Version: 05-15-19
+ * JavaScript file to create the clickable buttons for the chessboard
+ */
+
+/**
+ * JQuery using AJAX and JS to change the color of the chessboard upon
+ * clicking the buttons. Each button click sends a POST request to
+ * views/__init__.py to change the colors of the 8x8 chessboard grid
+ */
+$(document).ready(function() {
+    // When "Switch Tile Colors" button is clicked, send POST to View with clicked = true
+    $("#black-button").click(function() {
+        let input_data = {'clicked': true}
```

Comment:
line 15/22 - the `clicked` param isn't the most descriptive. Something like "color" might make more sense?